src/coordinator.rs

```diff
@@ -60,7 +60,9 @@ impl CoordinatorClient {
         transport.send_req(&req)
     }
 
-    // Get Spend transaction spending the vault with the given deposit outpoint.
+    /// Get Spend transaction spending the vault with the given deposit outpoint.
+    /// Beware that the spend transaction may be invalid and needs to be verified against
+    /// libbitcoinconsensus.
```

Comment:
I may have misunderstood libbitcoinconsensus usage, but it was to check that the sigs and the tx are valid and can pass the btc contensus and that the coordinator did not trick us with a fake spend that will never pass the mempool

Reply:
`libbitcoinconsensus` defines part of the consensus (and even more so of the standard Bitcoin Core relay policy). It can only do context-less checks. For instance, it would not be able to detect if the inputs of the Spend don't exist.

But if they committed to an invalid Spend.. It's their problem? It will be unspendable through the manager path so no harm can be done (assuming cosigning servers aren't compromised, which Spend things always do). They will have to Cancel all the Unvaults.